homeassistant/components/plugwise/strings.json

```diff
@@ -11,7 +11,10 @@
           "username": "Smile Username"
         },
         "data_description": {
-          "host": "Leave empty if using Auto Discovery"
+          "password": "The Smile ID printed on the label on the back of your Adam, Smile-T, or P1.",
+          "host": "The hostname or IP-address of your Smile. You can find it in your router or the Plugwise App. Check https://www.home-assistant.io/integrations/plugwise/#configuration for more help.",
```

Comment:
Normally we don't like URLs in translation strings, and we recommend using placeholders instead.
In this case,I think it is enough to remove it as it points to the regular integration documentation.

Reply:
Adjusted accordingly